Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -113,9 +114,9 @@ def incident_data_to_demisto_format(inc_data):
         'Status': properties.get('status'),
         'AssigneeName': properties.get('owner', {}).get('assignedTo'),
         'AssigneeEmail': properties.get('owner', {}).get('email'),
-        'Label': [{
-            'Name': label.get('name'),
-            'Type': label.get('type')
+        'Labels': [{
+            'Name': label.get('labelName'),
+            'Type': label.get('labelType')
```

Comment:
What's the expected output in the yaml?

Reply:
The yaml states a `Label` output, I'll change it. 

In `INCIDENT_HEADERS` (used for human-readable output), the header is  `Labels` though. Should they match or is it a BW compatibility issue if we change it now?

Reply:
`Label` it is